src/screen.test.js

```diff
@@ -1,7 +1,7 @@
 describe('Render screen', () => {
   it('Should contain Screen div', () => {
     cy.visit('http://localhost:3000');
-    cy.get('#debug-bar-fixture-1').click();
+    cy.get('select').select('slideshow2');
```

Comment:
It's better to use a class or ID as selector, since other 'select' elements might exist.